src/main/java/it/gov/pagopa/pu/registry/mapper/pagopa/RegistryMappingService.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.pu.registry.mapper.pagopa;
+
+import it.gov.pagopa.pu.registry.utils.RegistryUtils;
+import org.apache.commons.lang3.function.TriFunction;
+import org.springframework.data.util.StreamUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+@Service
+public class RegistryMappingService {
```

Comment:
Instead of introducing a new Service, let's make it abstract, using it as superclass for registry mappers.
This way we will introduce a strong dependency between these utility methods
```suggestion
public abstract class BaseRegistryMapper <T,R> {
```